tests/test_util_shadertoy.py

```diff
@@ -18,15 +18,15 @@ def force_offscreen():
         del os.environ["WGPU_FORCE_OFFSCREEN"]
 
 
-def test_shadertoy():
+def test_shadertoy_wgsl():
     # Import here, because it imports the wgou.gui.auto
```

Comment:
typo (was there earlier)
```suggestion
    # Import here, because it imports the wgpu.gui.auto
```